master/internal/experiment.go

```diff
@@ -484,25 +504,27 @@ func (e *experiment) isBestValidation(msg searcher.CompletedMessage) bool {
 	return isBest
 }
 
-func (e *experiment) updateState(ctx *actor.Context, state model.State) {
-	if wasPatched, err := e.Transition(state); err != nil {
+func (e *experiment) updateState(ctx *actor.Context, state model.State) (err error) {
+	var wasPatched bool
```

Comment:
I think with `err` already defined running `wasPatched, err := e.Transition` causes a listing error

Reply:
It is causing an error because you named returned value. You should revert that back first and then move the wasPatched back to its original code as well.

Reply:
yep, it was defined there to allow it to be used in the rest of the function for catching database errors. since I took out that part we can put this back also.